src/index.js

```diff
@@ -258,19 +260,22 @@ class IpfsRepo {
       getSize(this.datastore),
       getSize(this.keys)
     ])
-    let size = blocks.size
+    const size = blocks.size
       .plus(datastore)
       .plus(keys)
 
-    if (options.human) {
-      size = size.div(1048576)
-    }
     return {
       repoPath: this.path,
-      storageMax: storageMax,
+      storageMax: options.human
+        ? filesize(storageMax, { exponent: 3 }) // exponent 3 specifies the symbol GB for base 2
+        : storageMax,
       version: version,
-      numObjects: blocks.count,
-      repoSize: size
+      numObjects: options.human
+        ? blocks.count.toNumber()
+        : blocks.count,
+      repoSize: options.human
+        ? filesize(size, { exponent: 2 }) // exponent 2 specifies the symbol MB for base 2
```

Comment:
My opinion on this is not terribly strong but I guess it'd be nice to let the library figure out a suitable representation - if the repo size is 10KB, show 10KB instead of 0.01MB